analytics_data_api/v0/serializers.py

```diff
@@ -75,6 +75,43 @@ class Meta(object):
         )
 
 
+class ConsolidatedAnswerDistributionSerializer(ProblemResponseAnswerDistributionSerializer):
+    """
+    Serializer for consolidated answer distributions.
+    """
+
+    consolidated_variant = serializers.BooleanField()
+
+    class Meta(ProblemResponseAnswerDistributionSerializer.Meta):
+        fields = (
+            'course_id',
+            'module_id',
+            'part_id',
+            'correct',
+            'count',
+            'value_id',
+            'answer_value_text',
+            'answer_value_numeric',
+            'problem_display_name',
+            'question_text',
+            'variant',
+            'consolidated_variant',
+            'created'
+        )
+
+    # pylint: disable=super-on-old-class
+    def restore_object(self, attrs, instance=None):
```

Comment:
This is the only way to add a non model field to a Django ModelSerializer at the moment.  There's an open request to add a non_native_fields field and an ExtensibleModelSerializer to the Django Rest Framework, but it isn't in our version.